snap/info.go

```diff
@@ -31,6 +31,19 @@ import (
 	"github.com/ubuntu-core/snappy/timeout"
 )
 
+// FIXME: make this a struct and add a special unmarshal that will deal
+//        with both int and string
```

Comment:
Atually, I see where your idea comes from. This would make it impossible to show it with the wrong formatting. That's a great idea.